apps/hobbies-helsinki/src/domain/event/queryUtils.ts

```diff
@@ -31,18 +31,21 @@ const useSimilarEventsQueryVariables = (event: EventFields) => {
         .map((category) => category?.id) // collect ids
         .filter((id) => id != null) // remove nulls and undefined ones
         .join(), // make a string
-      [EVENT_SEARCH_FILTERS.MIN_AGE]: event.audienceMinAge ?? '',
-      [EVENT_SEARCH_FILTERS.MAX_AGE]: event.audienceMaxAge ?? '',
+      [EVENT_SEARCH_FILTERS.MIN_AGE]: event.audienceMinAge ?? '5',
+      [EVENT_SEARCH_FILTERS.MAX_AGE]: event.audienceMaxAge ?? '15',
```

Comment:
Is the default age limit 5–15 here intended for tests only? I mean `useSimilarEventsQueryVariables` -> `useSimilarEventsQuery` -> `SimilarEvents` -> `EventPageContainer` which is used in other than tests so this affects not only tests.

Reply:
oh yea. I want to remove my approval! :D